src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,66 @@
+package duke;
+
+import duke.commands.ByeCommand;
+import duke.commands.Command;
+import duke.exceptions.DukeException;
+import duke.managers.ParserManager;
+import duke.managers.StorageManager;
+import duke.managers.TaskManager;
+import duke.managers.UiManager;
+
+/**
+ * Initializes the Duke application and initiates a conversation with the user.
+ *
+ * @author Emily Ong Hui Qi
+ */
+public class Duke {
+
+    /**
+     * Name of the chatbot
+     */
+    private static final String NAME = "Duke";
+
+    /**
+     * The greeting message used by the chatbot when the application starts
+     */
+    private static final String MESSAGE_GREETING = String.format("Hello! I'm %s\nWhat can I do for you?", Duke.NAME);
+
+    /**
+     * The main event loop of the application.
+     *
+     * @param args Unused arguments
```

Comment:
```suggestion
     * @param args Unused arguments.
```

Should there be punctuation added? [relevant coding standard](https://se-education.org/guides/conventions/java/intermediate.html#:~:text=Punctuation%20behind%20each%20parameter%20description)

Reply:
Thanks!